pkgs/development/interpreters/lua-5/interpreter.nix

```diff
@@ -66,7 +74,8 @@ self = stdenv.mkDerivation rec {
     makeFlagsArray+=(${lib.optionalString stdenv.isDarwin "CC=\"$CC\""}${lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) " 'AR=${stdenv.cc.targetPrefix}ar rcu'"})
 
     installFlagsArray=( TO_BIN="lua luac" INSTALL_DATA='cp -d' \
-      TO_LIB="${if stdenv.isDarwin then "liblua.${version}.dylib" else "liblua.a liblua.so liblua.so.${luaversion} liblua.so.${version}"}" )
+      TO_LIB="${if stdenv.isDarwin then "liblua.${version}.dylib"
+                else "liblua.a" + (if (!staticOnly) then " liblua.so liblua.so.${luaversion} liblua.so.${version}" else "")}" )
```

Comment:
hum does the + apply to just liblua.a or the whole if then else ?

Reply:
Only to liblua.a. I added parentheses to make it clean.